plutus-tx-plugin/test/Budget/9.6/findIndexExpensive.budget.golden

```diff
@@ -1,2 +1,2 @@
-({cpu: 13963400
-| mem: 39440})
\ No newline at end of file
+({cpu: 13825400
+| mem: 38840})
```

Comment:
Barely any difference for `findIndex` and a big difference for `elem`:

```diff
- ({cpu: 10254990
- | mem: 32120})
+ ({cpu: 9426990
+ | mem: 28520})
```

and `all`:

```diff
- ({cpu: 10219630
- | mem: 32120})
+ ({cpu: 9391630
+ | mem: 28520})
```

The more I look at it, the more confused I become.